src/type/validate.js

```diff
@@ -169,15 +169,14 @@ function getOperationTypeNode(
 }
 
 function validateDirectives(context: SchemaValidationContext): void {
-  const directives = context.schema.getDirectives();
-  directives.forEach(directive => {
```

Comment:
Is there a concrete reason to make this change? `.forEach` is how most things are done in this library, and I'd prefer to maintain consistency.

Reply:
@mjmahone I used `for of` in all new code I wrote for readability and performance reasons but I fully agree with consistency argument so I created #1423 where I converted all other instances of `Array.forEach`.